pkg/tfvars/libvirt/libvirt.go

```diff
@@ -83,7 +83,14 @@ func TFVars(sources TFVarsSources) ([]byte, error) {
 	}
 
 	if sources.MasterConfig.Volume.VolumeSize != nil {
-		cfg.MasterDiskSize = sources.MasterConfig.Volume.VolumeSize.String()
+		// As per https://github.com/hashicorp/terraform/issues/3287 the
+		// master disk size needs to be specified as the number of bytes.
+		diskSizeInBytes, converted := sources.MasterConfig.Volume.VolumeSize.AsInt64()
```

Comment:
So this is just a validation to make sure if user try to change master disk size it should be number of bytes instead of using units for measures, We are using this in snc but always used bytes https://github.com/code-ready/snc/blob/master/snc.sh#L139 so validation should be good 👍🏼

Reply:
This is not just validation, right? This is conversion to a form that terraform can handle. The user can still use, for example, 1024M in the Machine and this code will convert that value to 1048576 (or whatever the converted value is) for consumption by terraform.

Reply:
Correct, it's ensuring that bytes are output for terraform since that's all it understands. In the generated manifests it's still possible to specify whatever is supported by kube's Quantity type.

Reply:
Ah my bad, I thought we are trying to first convert whatever user provide and then use this as validation but this is really good to see it support quantity type.